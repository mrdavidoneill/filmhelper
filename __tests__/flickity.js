import React from "react";
import { render } from "@testing-library/react";
import Carousel from "../components/flickity";

describe("Carousel", () => {
  it("should match the snapshot", () => {
    const images = ["image1.jpg", "image2.jpg"];
    const setCurrentIndex = jest.fn();
    const initialIndex = 0;

    const { container } = render(
      <Carousel
        images={images}
        setCurrentIndex={setCurrentIndex}
        initialIndex={initialIndex}
      />
    );

    expect(container).toMatchSnapshot();
  });
});
