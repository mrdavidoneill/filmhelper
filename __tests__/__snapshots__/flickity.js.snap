// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel should match the snapshot 1`] = `
<div>
  <div
    class="bg-black cursor-pointer outline-white"
  >
    <div
      class=" flex flex-grow mx-2 h-[35vh] w-[30vh] max-w-sm bg-cover bg-center"
      style="background-image: url(image1.jpg);"
    />
    <div
      class=" flex flex-grow mx-2 h-[35vh] w-[30vh] max-w-sm bg-cover bg-center"
      style="background-image: url(image2.jpg);"
    />
  </div>
</div>
`;
